Sources/Sentry/include/SentryUIApplication.h

```diff
@@ -41,6 +42,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSArray<UIScene *> *)getApplicationConnectedScenes:(UIApplication *)application
     API_AVAILABLE(ios(13.0), tvos(13.0));
+
+/**
+ * Retrieve the topmost view controllers currently presented in the app.
+ * In the case of a multi-screen iPad app, it will contain the most relevant
+ * view controller for each screen. If the topmost view controller
+ * is a split view controller, all of its sides are reported.
+ */
+- (nullable NSArray<UIViewController *> *)relevantViewControllers;
```

Comment:
`m`: I think this method can now be internal to `SentryUIApplication`. No other class is using it, they all use `relevantViewControllersNames`.